index.js

```diff
@@ -304,11 +304,17 @@ module.exports = function StateProvider(makeRenderer, rootElement, stateRouterOp
 	}
 
 	function makePath(stateName, parameters, options) {
+		function guaranteePreviousState() {
+			if (!lastStateStartedActivating.get().name) {
+				throw new Error('makePath required a previous state to exist, and none was found')
+			}
+		}
```

Comment:
Getting into semantics here, but does `guaranteePreviousState()` actually gaurantee that a previous states exists? I feel like it's more of an assertion.